qiita_app/lib/widgets/article_container.dart

```diff
@@ -0,0 +1,93 @@
+import 'package:flutter/material.dart';
+import 'package:qiita_app/constants/app_colors.dart';
+import 'package:qiita_app/models/article.dart';
+
+class ArticleContainer extends StatelessWidget {
+  const ArticleContainer({
+    Key? key,
+    required this.article,
+  }) : super(key: key);
+
+  final Article article;
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      children: [
+        Padding(
+          padding: const EdgeInsets.symmetric(horizontal: 16, vertical: 8),
+          child: Row(
+            crossAxisAlignment: CrossAxisAlignment.start,
+            children: [
+              CircleAvatar(
+                backgroundImage: NetworkImage(article.userIconUrl),
+                radius: 19,
+              ),
+              const SizedBox(width: 8),
+              Expanded(
+                child: Column(
+                  crossAxisAlignment: CrossAxisAlignment.start,
+                  children: [
+                    Text(
+                      article.title,
+                      maxLines: 2,
+                      overflow: TextOverflow.ellipsis,
+                      style: const TextStyle(
+                        fontSize: 14,
+                        fontWeight: FontWeight.bold,
+                        color: AppColors.black,
+                      ),
+                    ),
+                    Row(
+                      children: [
+                        Text(
+                          '＠${article.userName}',
+                          style: const TextStyle(
+                            fontSize: 12,
+                            color: AppColors.secondary,
+                          ),
+                        ),
+                        const SizedBox(width: 4),
+                        Text(
+                          '投稿日:${article.postedDate}',
+                          style: const TextStyle(
+                            fontSize: 12,
+                            color: AppColors.secondary,
+                          ),
+                        ),
+                        const SizedBox(width: 4),
+                        Text(
+                          'いいね：${article.likesCount}',
+                          style: const TextStyle(
+                            fontSize: 12,
+                            color: AppColors.secondary,
+                          ),
```

Comment:
textのスタイルを再利用して短くするといいんじゃないかなと思いまいた！

```suggestion
                      final TextStyle commonTextStyle = TextStyle(
  fontSize: 12,
  color: AppColors.secondary,
);

Text(
  '＠${article.userName}',
  style: commonTextStyle,
),
Text(
  '投稿日:${article.postedDate}',
  style: commonTextStyle,
),
Text(
  'いいね：${article.likesCount}',
  style: commonTextStyle,
),
```